app/views/components/forms/edit/question_types/_checkbox_option.html.erb

```diff
@@ -1,17 +1,16 @@
 <div class="question-option list-group-item" id="<%= dom_id(option) %>" data-id="<%= option.id %>">
   <%= check_box_tag("#{question.answer_field}_#{(index + 1).to_s}", index + 1, nil, { value: option.text , class: "usa-checkbox__input" }) %>
   <%= label_tag(nil, for: "#{question.answer_field}_#{(index + 1).to_s}", class: "usa-checkbox__label") do %>
-    <span class="question-option-view"><%= option.text %></span>
+    <span class="question-option-view"><%= option.text %>(<%= option.value %>)</span>
     <span class="question-option-edit">
-      <input type="text" value="<%= option.text %>" class="usa-input display-inline" data-href="<%= admin_form_question_question_option_path(question.form, question, option) %>" />
+      <input type="text" value="<%= option.text %>" class="usa-input display-inline" />&nbsp;&nbsp;
+      <input type="text" value="<%= option.value %>" class="usa-input display-inline"/>&nbsp;&nbsp;
+      <%= link_to admin_form_question_question_option_path(question.form, question, option), style: "text-decoration: none;", class: "delete button form-save-question-option"  do %>
```

Comment:
Move this inline style to .css.

Inline styles are discouraged with stricter browser security policies.